src/rgw/rgw_rest_s3.cc

```diff
@@ -4516,6 +4516,9 @@ RGWOp *RGWHandler_REST_Obj_S3::op_post()
 
   if (s->info.args.exists("uploads"))
     return new RGWInitMultipart_ObjStore_S3;
+  
+  if (s->info.args.exists("select-type"))
+    return new RGWS3Select;
```

Comment:
nit: better to keep naming conventions of the ops. e.g. `RGWSelectObj_ObjStore_S3`

Reply:
ok